docs/docs/widgets/timer.md

```diff
@@ -2,90 +2,92 @@
 id: timer
 title: Timer
 ---
+
 # Timer
 
-The **Timer** widget lets users to count timer both upward and downward
+The **Timer** component allows users to track time by counting both upward and downward. It's useful for tasks like setting countdowns, tracking elapsed time, or timing events
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Properties
 
-| <div style={{ width:"100px"}}> Property </div>     | <div style={{ width:"100px"}}> Description </div> |
-|:----------- |:----------- | 
-| Default value | It specifies the initial value of timer. Format is: `HH.mm.ss.SS`. |
-| Timer type | It specifies If its a upward or downward counter. Select `Count Up` or `Count Down` from the dropdown or you can click on `Fx` to programmatically define the values `countUp` or `countDown`. |
+| <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div>                                                                                                                                                        |
+| :--------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Default value                                  | It specifies the initial value of timer. Format is: `HH.MM.SS.MS`.                                                                                                                                       |
+| Timer type                                     | It specifies If its a upward or downward counter. Select **Count Up** or **Count Down** from the dropdown or you can click on **Fx** to programmatically define the values **countUp** or **countDown**. |
```

Comment:
Hey, the f in the fx should be small and not Fx, please check for this in the whole doc and both the version.